backend/clickhouse/logs_test.go

```diff
@@ -57,6 +57,33 @@ func assertCursorsOutput(t *testing.T, edges []*modelInputs.LogEdge, expectedCur
 	}
 }
 
+func TestBatchWriteLogRows(t *testing.T) {
+	ctx := context.Background()
+	client, teardown := setupTest(t)
+	defer teardown(t)
+
+	now := time.Now()
+
+	logRow := NewLogRow(
+		LogRowPrimaryAttrs{},
+		WithSource(modelInputs.LogSourceFrontend),
+		WithProjectIDString("1"),
+		WithTimestamp(now),
+	)
+	rows := []*LogRow{
+		logRow,
+	}
+
+	assert.NoError(t, client.BatchWriteLogRows(ctx, rows))
+
+	payload, err := client.ReadLogs(ctx, 1, modelInputs.LogsParamsInput{
+		DateRange: makeDateWithinRange(now),
+	}, Pagination{})
+	assert.NoError(t, err)
+	assert.Len(t, payload.Edges, 1)
+	assert.Equal(t, modelInputs.LogSourceFrontend.String(), *payload.Edges[0].Node.Source)
```

Comment:
I just wanted to confirm that changing the type on `Source`:
https://github.com/highlight/highlight/blob/d8ee3158b7599a8e1fcecde73e3d701e4d84b274/backend/clickhouse/log_row.go#L47

still correctly writes (and reads) out the expected data.